Library/src/Multiplayer/SequenceHierarchy.cpp

```diff
@@ -0,0 +1,59 @@
+#include "CSP/Multiplayer/SequenceHierarchy.h"
+
+#include "CSP/Systems/Sequence/Sequence.h"
+
+#include <string>
+
+namespace csp::multiplayer
+{
+
+const SequenceHierarchy& csp::multiplayer::SequenceHierarchyResult::GetSequenceHierarchy() const
+{
+	return SequenceHierarchy;
+}
+
+void SequenceHierarchyResult::OnResponse(const csp::services::ApiResponseBase* ApiResponse)
+{
+}
+
+common::String CreateSequenceKey(common::Optional<uint64_t> ParentId, const common::String& SpaceId)
+{
+	common::String Key = SequenceHierarchyName + ":" + SpaceId;
+
+	if (ParentId.HasValue())
+	{
+		Key += ":" + systems::SequenceIdPrefix + std::to_string(*ParentId).c_str();
```

Comment:
Shouldn't we be checking these for invalid (% / ) characters?

Reply:
ParentId is a uint64_t, and SpaceId is a hex string, so this would never happen unless a user passes in a random string for the SpaceId

Reply:
This function isn't used internally by the SequenceHierachy, so there is no way they could pass in an invalid space

Reply:
This function is only exposed for testing purposes